bridges/deployments/local-scripts/relay-headers-rococo-to-wococo.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I suppose these changes don't come via `subtree` update, IMHO we shouldn't be mixing adding `bridges` code and `polkadot` code in one PR. The point of `bridges` code being part of the repository is to make it easy for **polkadot devs** to fix any substrate inconsistencies, not to add features.